packages/syntax-core/src/Avatar/Avatar.module.css

```diff
@@ -3,24 +3,25 @@
   background-size: cover;
   background-repeat: no-repeat;
   border-radius: 50%;
+  border: 1px solid #fff;
 }
 
 .sm {
-  width: 24px;
-  height: 24px;
+  width: 22px;
```

Comment:
@zprong It's not very clear why we use `22px` instead of `24px` - I know it's the calculation of `24px` (image width) - the border on each side (`2 * 1px`) resulting in `22px`.

Instead, we could use CSS variables to make this more explicit:

```css
:root {
  --syntax-avatar-border-width: 1px;
}

.avatar {
  background-position: center;
  background-size: cover;
  background-repeat: no-repeat;
  border-radius: 50%;
  border: var(--syntax-avatar-border-width) solid #fff;
}

.sm {
  width: calc(24px - var(--syntax-avatar-border-width) * 2);
  height: calc(24px - var(--syntax-avatar-border-width) * 2);
}

.md {
  width: calc(40px - var(--syntax-avatar-border-width) * 2);
  height: calc(40px - var(--syntax-avatar-border-width) * 2);
}

.lg {
  width: calc(72px - var(--syntax-avatar-border-width) * 2);
  height: calc(72px - var(--syntax-avatar-border-width) * 2);
}

.xl {
  width: calc(128px - var(--syntax-avatar-border-width) * 2);
  height: calc(128px - var(--syntax-avatar-border-width) * 2);
}
```

This way, if we ever change the border width to e.g. `2px`, we don't have to do the calculations again ourselves.

Reply:
Brilliant! I'll incorporate this, thanks Christian!